docs/autorefresh.asciidoc

```diff
@@ -6,14 +6,16 @@ When a refresh interval is set, it is displayed to the left of the Time Filter i
 
 To set the refresh interval:
 
-. Click the *Time Filter* image:images/TimeFilter.jpg[Time Filter] in the upper right corner of the menu bar.
+. Click the *Time Filter* image:images/TimeFilter.jpg[Time Filter].
```

Comment:
All of the images in this section seem to be from the old design with the dark toolbar.